courses/free-courses-bn.md

```diff
@@ -16,7 +16,9 @@
 * [IDE and editors](#ide-and-editors)
 * [Java](#java)
 * [JavaScript](#javascript)
+    * [AngularJS](#angularjs)
```

Comment:
The heading is Angular, not AngularJS, so we got this wrong. I prefer `#angular`, although I'm not positive this is correct. either way, the heading and the index need to match.

Reply:
@eshellman Did the necessary changes. Is this fine now?